WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemDataSource.kt

```diff
@@ -10,26 +10,21 @@ import org.wordpress.android.fluxc.model.PostSummary
 import org.wordpress.android.fluxc.model.SiteModel
```

Comment:
I just noticed we are still importing `PostSummary`. Lint is not complaining since the reference is used in the class-level comment.

Reply:
Fixed in a5918ccb021f5c8b7bc58c2a60f6eaba9fd58b82.